facebook.js

```diff
@@ -1,82 +1,45 @@
-const emoji_map = {
-  "1 person": "👤",
-  "2 people": "👥",
-  "3 people": "👥",
-  "4 people": "👥",
-  "5 people": "👥",
-  "6 people": "👥",
-  "7 people": "👥",
-  "baby": "🍼",
-  "beach": "🏖",
-  "beard": "👴",
-  "bedroom": "🛏",
-  "bicycle": "🚲",
-  "camera": "📷",
-  "car": "🚗",
-  "cat": "😺",
-  "child": "👦",
-  "christmas tree": "🎄",
-  "close-up": "👀",
-  "closeup": "👀",
-  "cloud": "☁️",
-  "concert": "🎤",
-  "crowd": "👥",
-  "dancing": "💃",
-  "dessert": "🍰",
-  "dog": "🐶",
-  "drink": "🍹",
-  "eating": "🍽",
-  "eyeglasses": "👓",
-  "fireworks": "🎆",
-  "flower": "🌻",
-  "food": "🍎",
-  "glasses": "🕶",
-  "golf": "🏌️‍",
-  "grass": "🍃",
-  "hat": "👒",
-  "indoor": "🏠",
-  "living room": "🏠",
-  "meme": "👍",
-  "mountain": "🌋",
-  "nature": "🏞",
-  "night": "🌃",
-  "ocean": "🌊",
-  "office": "💼",
-  "one or more people": "👥",
-  "outdoor": "🚵",
-  "people eating": "🍽",
-  "people on stage": "🎤",
-  "people playing musical instruments": "🎸",
-  "people playing sport": "🏀",
-  "people sitting": "⑁",
-  "people smiling": "😂",
-  "people standing": "🕴",
-  "phone": "📱",
-  "plant": "🌿",
-  "playing a musical instrument": "🎸",
-  "selfie": "🤳",
-  "shoes": "👡",
-  "sitting": "⑁",
-  "sky": "☀️",
-  "skyscraper": "🏙",
-  "sleeping": "😴",
-  "smiling": "😋",
-  "snow": "❄️",
-  "standing": "🕴",
-  "stripes": "📶",
-  "suit": "🕴",
-  "sunglasses": "🕶",
-  "swimming": "🏊",
-  "table": "🍽",
-  "text": "🔠",
-  "tree": "🌴",
-  "twilight": "🌃",
-  "water": "💧",
-  "wedding": "💒"
+var locale = (Array.from(document.body.classList).find(cls => cls.match(/^Locale_/)));
+let emoji_map = {};
+var TAG_PREFIX;
+  var xhr = new XMLHttpRequest();
+if (locale == 'Locale_es_LA') {
+ TAG_PREFIX = "La imagen puede contener: ";
+xhr.open('GET', chrome.extension.getURL('/locales/es/messages.txt'), true);
```

Comment:
Instead of XHR, you could use [`fetch` API](https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch), which is slightly neater. Something like this:

```javascript
let req;
if (locale === 'Locale_en_US') {
    req = new Request('locales/en_US.json');
}

if (req) {
    fetch(req).then(function(response) {
        return response.json();
    }).then(function(response) {
        /* start to observe */
    });
}
```

Also, uneven indentation.